submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/MultipartFileSubmissionApiTest.kt

```diff
@@ -206,14 +213,89 @@ internal class MultipartFileSubmissionApiTest(
             val response = webClient.submitSingle(submission, XML, listOf(fileList, tempFolder.createFile("File3.txt")))
             assertThat(response).isSuccessful()
             assertSubmissionFiles("S-TEST3", "File3.txt")
+            fileList.delete()
+        }
+
+        @Test
+        fun `JSON submission with TSV file list`() {
```

Comment:
I will not categorize these ones as part of "multipart test" considering that we group by feature, maybe file list itest?

Reply:
Done